hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -608,9 +607,10 @@ static void xpadneo_update_battery(struct xpadneo_devdata *xdata, u8 value)
 		break;
 	default:
 		xdata->psy_desc.type = POWER_SUPPLY_TYPE_UNKNOWN;
+		break;
```

Comment:
I'm undecided about this. I don't think "default" should have an explicit break. You either put it last to have a fallback, or you put it first to have a common initializer.

Reply:
I believe fall through in any case is bad practice, and whatever is trying to be achieved with it can be achieved in other, simpler ways. It always comes as a surprise when I'm reading through code.

Reply:
Okay, then let's accept this change.